docs/source/get-involved.rst

```diff
@@ -0,0 +1,46 @@
+Get involved
+============
+
+Join the dask mailing list!
+----------------------
+
+We welcome you to join the `dask mailing list <https://groups.google.com/a/continuum.io/forum/#!forum/blaze-dev>`_, where users and the project developers:
+
+* ask and answer questions
+* discuss ways to use dask
+* collaborate on solutions and approaches, and 
+* explore possible new features.
+
+
+Chat with other users
+---------------------
+
+Discuss and ask questions here:
+
+* `StackOverflow <http://stackoverflow.com/questions/tagged/dask>`_ using the #dask tag
+* The `gitter chat room <https://gitter.im/blaze/dask>`_
+
+
+Report a bug
+------------
+
+Open issues with dask are tracked on GitHub at https://github.com/blaze/dask/issues. If you find a bug, please:
+
+1.  Search issues to see if yours has already been reported. If no one has,
+2.  Create a new issue to report it!
+
+
+Get paid support
+---------------------
+
+Paid training and support options:
```

Comment:
It might be worth explicitly prompting this by saying that dask is primarily developed by developers employed by Continuum and that Continuum offers the services below.  Otherwise it makes dask sound like a commercial entity.  

I very much want to avoid the impression that dask is a commercial entity.